src/js/views/patients/sidebar/action/action-sidebar-action_views.js

```diff
@@ -177,19 +183,28 @@ const ActionView = View.extend({
     this.showChildView('save', new SaveView({ model: this.clonedAction }));
   },
   onSave() {
-    this.getRegion('save').empty();
+    this.stopEditingDetails();
   },
   onCancel() {
     this.showEditForm();
   },
   showEditForm() {
     this.cloneAction();
-    this.listenTo(this.clonedAction, 'change:details', this.showSave);
 
-    this.getRegion('save').empty();
+    this.listenTo(this.clonedAction, 'change:details', () => {
```

Comment:
can we make this line `this.listenTo(this.clonedAction, 'change:details', this.startEditingDetails);` that's the flag and the this.showChildView line from `showSave`?

Reply:
Nice, I like it. Just pushed that change 👇